modules/virt-copying-the-ssh-command.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+// * virt/vm_networking/virt-using-the-default-pod-network-with-virt.adoc
+
+:_content-type: PROCEDURE
+[id="virt-copying-the-ssh-command_{context}"]
+= Copying the SSH Command from the web console
+
+Copy the SSH command of your virtual machine (VM) from the web console for use in the OpenShift CLI (`oc`).
+
+.Prerequisites
+* A configured virtual machine.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Virtualization* -> *VirtualMachines* from the side menu.
+. Use the search and filters to find your project in the VM list.
```

Comment:
The "project" should be "virtual machine" here.

Reply:
+1 @gouyang 
Thank you.